libs/blocks/marquee/marquee.css

```
@@ -17,30 +17,28 @@
   margin-bottom: 0;
 }
 
-.marquee.light, .marquee.mobile-light, .marquee.quiet, .marquee.inline {
+.marquee:is(.light, .mobile-light, .quiet, .inline) {
   color: var(--text-color);
 }
 
-.marquee.mobile-dark {
+.marquee:is(.mobile-dark, .quiet.dark) {
```

```suggestion
.marquee:is(.mobile-dark, .quiet.dark) {
```
I would need to test this, but since the regular marquee supports .dark .marquee theme inheritance. We might want to add some selectors for this for when this marquee is in a section.style.dark? 🤷